src/browser-frontend/views/browser/chrome/tabbar/tab.css

```diff
@@ -12,9 +12,17 @@ specific language governing permissions and limitations under the License.
 
 .tab {
   overflow: hidden;
+  width: var(--theme-tab-open-width);
```

Comment:
No need for the `width` rule here since we already have it in the `.initial` class.